sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -593,6 +593,7 @@ private[sbtplugin] object ScalaJSPluginInternal {
          */
         val config = RunConfig()
           .withLogger(sbtLogger2ToolsLogger(log))
+          .withEnv((envVars in run).value)
```

Comment:
Should this be `run / envVars`?

Reply:
No, the sbt plugin compiles against the API of sbt 1.0.0, still to this day.

Reply:
Sure, wasn't sure of the minimum version.